xrdcp_recursive.py

```diff
@@ -2,6 +2,10 @@
 
 """This module will recursively xrdcp (XRootD copy) files from a local directory to a XRootD endpoint.
 
+Note to users: This is not to be used for xrdcp'ing files from an XRootD source to a local directory,
+nor from an XRootD source to an XRootD endpoint. For those kinds of transfers, XRootD already supports
+them with the command `xrdcp -r'.
```

Comment:
I'm not really sure that `xrdcp -r` works. Have you tested it? Do you know it works? If so, I think its fine to say this. Otherwise I would remove the comment starting at "For those kinds of transfers..."

Reply:
Nah, I trust you.